website_event_compassion/controllers/events_controller.py

```diff
@@ -25,9 +25,17 @@ class EventsController(PaymentFormController):
 
     @http.route('/events/', auth='public', website=True)
     def list(self, **kwargs):
-        events = request.env['crm.event.compassion'].search([
+        today = fields.Date.to_string(datetime.today())
+        started_events = request.env['crm.event.compassion'].search([
             ('website_published', '=', True),
+            ('start_date', '<', today),
```

Comment:
I think you can achieve the same result by just removing the filter on start_date, then you have all your events and don't need your second search.
```suggestion
```